src/libpriv/rpmostree-passwd-util.c

```diff
@@ -646,13 +646,22 @@ rpmostree_passwd_migrate_except_root (int            rootfs_dfd,
 {
   GLNX_AUTO_PREFIX_ERROR ("passwd migration", error);
   const char *name = kind == RPM_OSTREE_PASSWD_MIGRATE_PASSWD ? "passwd" : "group";
+  struct stat stbuf;
+  const char *etc_path;
+
+  if (!glnx_fstatat_allow_noent (rootfs_dfd, "usr/etc", &stbuf, 0, error))
+    return FALSE;
+  if (errno == ENOENT)
+    etc_path = "etc";
```

Comment:
But this should never happen, right? We've already moved `/etc` to `/usr/etc`, so it'd be an error to *not* find it there, no?

Reply:
Updated for everything except this...I still need to investigate this bit more.

Reply:
OK, upon further investigation I think you're right.  I haven't yet rebased unified core on top of this to verify but if there's an issue with that we can fix it later.